lib/stagger/aggregator.rb

```diff
@@ -33,9 +33,17 @@ def value(key, value, weight = 1, tags = {})
     end
 
     def delta(key, value, tags = {})
-      incr(key, @deltas[to_key(key, tags)].delta(value))
+      delta_incr(key, @deltas[to_key(key, tags)].delta(value))
+    end
+
+    # We don't want to use the `incr` method for Deltas
+    # Delta values may be 0, unlike counters, where
+    # incrementing makes no sense for 0 values
+    def delta_incr(key, value, tags = {})
```

Comment:
Should this method be either private, or (if there's only the one caller) just inlined in the `delta` method?